README.md

```diff
@@ -18,4 +18,5 @@ in the `cEP-0000.md` document.
 | [cEP-0012](cEP-0012.md) | coala's Command Line Interface     | This cEP describes the design of coala's Command Line Interface (the action selection screen).                                                                                                                                                        |
 | [cEP-0013](cEP-0013.md) | Cobot Enhancement and porting      | This cEP describes about the new features that are to be added to cobot as a part of the [GSoC project](https://summerofcode.withgoogle.com/projects/#4913450777051136).                                                                              |
 | [cEP-0014](cEP-0014.md) | Generate relevant coafile sections | This cEP proposes a framework for coala-quickstart to generate more relevant `.coafile` sections based on project files and user preferences.                                                                                                         |
+| [cEP-0015](cEP-0015.md) | corobo Enhancement                 | This cEP describes the details of enhancement for corobo in terms of security, tests and configurability as part of the [GSoC'18 Project](https://summerofcode.withgoogle.com/projects/#6603667076022272).                                            |
```

Comment:
For the consistency, it will be better to make it: `as part of the [GSoC'18 Project]` --> `as a part of the [GSoC'18 project]`